mifosng-android/src/main/res/values/strings.xml

```diff
@@ -567,6 +578,33 @@
     <string name="failed_to_fetch_center_info">Failed to fetch center info</string>
     <string name="generic_failure_message">Request failed. Please try again.</string>
 
+    <!-- Placeholder strings-->
+    <string name="row_collections_list_group_client_clientId">237</string>
+    <string name="row_collections_list_group_client_clientName">Raj Kumar Kaushikl</string>
+    <string name="row_savings_transaction_item_date">May 24,2014</string>
+    <string name="row_loan_transaction_item_date">May 24,2014</string>
+    <string name="row_savings_transaction_item_amount">2500 INR</string>
+    <string name="row_loan_transaction_item_amount">2500 INR</string>
+    <string name="row_loan_repayment_schedule_date">12-02-2014</string>
+    <string name="row_loan_repayment_schedule_id">200005</string>
+    <string name="row_loan_list_item_loanID">1</string>
+    <string name="view_account_accordian_section_value">0</string>
+    <string name="row_group_list_item_staff_name">M, Mary</string>
+    <string name="row_group_list_item_group_name">Santa Maria Group 1</string>
+    <string name="row_sync_payload_error_message">Error Message</string>
+    <string name="row_loan_transaction_item_detail_ID">09214</string>
+    <string name="row_client_list_item_client_name">460</string>
+    <string name="fragment_generate_collection_sheet_date">14 July 2014</string>
+    <!--End of Placeholder strings-->
+
+    <!--To do -->
+    <string name="view_account_accordian_section_to_do">{md-add-circle-outline}</string>
```

Comment:
This is not a placeholder, it is a real icon that will be rendered on runtime. It should be added as a hardcoded text.